playwright/async_api/_context_manager.py

```diff
@@ -27,15 +27,15 @@ def __init__(self) -> None:
         self._connection: Connection
 
     async def __aenter__(self) -> AsyncPlaywright:
+        loop = asyncio.get_running_loop()
         self._connection = Connection(
             None,
             create_remote_object,
-            PipeTransport(asyncio.get_event_loop(), compute_driver_executable()),
+            PipeTransport(loop, compute_driver_executable()),
         )
-        loop = asyncio.get_running_loop()
         self._connection._loop = loop
         loop.create_task(self._connection.run())
-        playwright_future = asyncio.create_task(
```

Comment:
It does by default use the current running loop, so I wouldn't change `asyncio.create_task` to `loop.create_task` or do I miss something?

Reply:
It does but this way it is consistent with `loop.create_task(self._connection.run())` or WDYT?

Reply:
convinced. SGTM. :+1: